_posts/2022-02-01-newpipe-0.21.16-hotfix.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+layout: post
+title: "NewPipe 0.21.16 released: Do not panic! We fixed YouTube."
+short: "NewPipe 0.21.16 released"
+date: 2022-02-01 09:00:00 + 01:00
+categories: [pinned, release]
+author: <a href="https://github.com/poolitzer">@Poolitzer</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Hello, did you order a hot fix update? Because we have one here, it is steaming and ready.
+
+<!-- more -->
+
+## YouTube broke.
+
+We are very sorry about this, but YouTube changed a parameter in their video player decryption and the code we created to decrypt the input from YouTube broke. [@AbduAmeen](https://github.com/AbduAmeen) created a fix and [@TobiGr](https://github.com/TobiGr) worked with them to get it in the Extractor. Thanks to both of them for doing this so quickly.
```

Comment:
Please don't forget @SebastianSong who also provided big parts of the code

Reply:
did not see them in the PR, sorry. added,